src/test/java/Homework18.java

```diff
@@ -0,0 +1,21 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest{
+    @Test
+    public void playSong() {
+        login("dmytro.kravchuk@testpro.io", "Fr440003");
+//        WebElement playSongButton = driver.findElement(By.cssSelector("[data-testid='play-btn']"));
+//     playSongButton.click();
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.